### DISCUSSION TITLE
请问我正尝试联机幸福工厂,但出现了一些问题

### DISCUSSION THREAD

**Discussion by tlatixat** [natfrp/wiki#390] [使用问题]
rt,我按照官方教程开好端口与服务器,确认内网电脑可以连接上服务器.
接着按照官方提示端口与udp协议开好隧道,得到ip与端口.
但是外网的伙伴无法用ip和端口链接到内网的服务器,网站流量监控无反应.
开好隧道后测试过,ping是通的.

**Comment by tlatixat** [comment]
刚才用telnet ping了下隧道开通的端口,显示的是连接失败

**Comment by Cassy-Lee** [comment]
确定要用UDP协议吗？
